exercises/rational-numbers/canonical-data.json

```diff
@@ -422,6 +422,30 @@
           "expected": [1, 1]
         }
       ]
+    },
+    {
+      "uuid": "bd1eab16-7603-4a77-ae4e-23c930fe39cd",
+      "description": "Ordering",
+      "property": "<",
```

Comment:
In the [diffie-hellman exercise](https://github.com/exercism/problem-specifications/blob/master/exercises/diffie-hellman/canonical-data.json#L31), we experimented with using a textual description of the property. This might be useful here too.

Reply:
Interesting, didn't know about that exercise. Do you think that format would be better here? I feel like it'd need special handling in generators either way, so I'm not sure if a description like in diffie-hellman has an advantage? It would likely need the comment anyway for extra explanations.

Reply:
It would definitely need the comment. I do think a textual property might be somewhat easier to understand. What do you think @SleeplessByte?

Reply:
Special handling is required either way (see linked-list); I think written out properties are more consistent with the other exercises.

Wouldn't care here 🤷🏽‍♂️🤗

Reply:
Okay, I'll write it out (that'll also make the schema check happy)